addons/im_livechat/i18n/im_livechat.pot

```diff
@@ -1868,8 +1870,8 @@ msgid "User's raw answer"
 msgstr ""
 
 #. module: im_livechat
-#. odoo-python
 #. odoo-javascript
+#. odoo-python
```

Comment:
unwanted diff

Reply:
It's generated by the tool, we shouldn't mess with it.